chapters/playframework/index.md

```diff
@@ -0,0 +1,546 @@
+---
+layout: default
+project: playframework
+title: Play Framework
+chapter: true
+---
+
+![play_full_color](images/play.png)
+
+# Play Framework
```

Comment:
You can change both. The Jekyll title is the one we show in the menu, the `# Title` is just what shows above the page. In both cases you may do something more "hip" than plainly the name of the project